src/stacks-hierarchy/Root_PurchaseOverviewScreen/use-offer-defaults.ts

```diff
@@ -59,6 +60,10 @@ export function useOfferDefaults(
     ],
   );
 
+  const selectableUserProfiles = userProfiles.filter((u) =>
+    selection.preassignedFareProduct.limitations.userProfileRefs.includes(u.id),
+  );
```

Comment:
Should use the `getSelectableUserProfiles` util from purchase-selection instead which also handles empty `limitations`.